packages/@okta/vuepress-site/use_cases/event_hooks/index.md

```diff
@@ -13,18 +13,28 @@ Event hooks are outbound calls from Okta, sent when specified events occur in yo
+Event hooks are Okta's implementation of the industry concept of webhooks. Okta's event hooks are related to, but different from, Okta [inline hooks](/use_cases/inline_hooks/): event hooks are meant to deliver information about events that occurred, not offer a way to affect execution of the underlying Okta process flow. Also, event hooks are asynchronous calls, meaning that the process flow that triggered the event hook continues without stopping or waiting for any response from your external service.
 
-You can have a maximum of 10 active and verified event hooks in your org at any point. Each event hook can be configured to deliver multiple event types.
+Before the introduction of event hooks, polling the [System Log API](/docs/api/resources/system_log/) was the only method your external software systems could use to detect the occurrence of specific events in your Okta org; event hooks provide an Okta-initiated push notification.
+
+You can have a maximum of 10 active and verified event hooks set up in your org at any time, and each event hook can be configured to deliver multiple event types.
 
 ## Which Events are Eligible?
 
-During the initial configuration procedure for an event hook, you specify which event types you want the event hook to deliver. The event types that can be specified are a subset of the event types that the Okta System Log captures. To see the list of event types currently eligible for use in event hooks, query the Event Types catalog with the query parameter `event-hook-eligible`:
+During the initial configuration procedure for an event hook, you specify which event types you want the event hook to deliver. The event types that can be specified are a subset of the event types that the Okta System Log captures.
+
+To see the list of event types currently eligible for use in event hooks, query the Event Types catalog with the query parameter `event-hook-eligible`:
 
 [https://developer.okta.com/docs/api/resources/event-types/?q=event-hook-eligible](/docs/api/resources/event-types/?q=event-hook-eligible)
 
+For general information on how Okta encapsulates events, see the [System Log API](/docs/api/resources/system_log/) documentation.
+
+Examples of available types of events include user lifecycle changes, the completion by a user of a specific stage in an Okta process flows, and changes in Okta objects. You could configure an event hook, for example, to deliver notification of user deactivation events. You could use this to trigger processes you need to execute internally every time a user is deactivated, like updating a record in an HR system, creating a ticket in a support system, or generating an email message.  
```

Comment:
flows to "flow" in first sentence
"a" notification in the second sentence

Examples of available types of events include user lifecycle changes, the completion by a user of a specific stage in an Okta process flow, and changes in Okta objects. You could configure an event hook, for example, to deliver notification of user deactivation events. .....

Reply:
thanks for catching, have corrected, but for "notification', decided to change it to a plural, so no "a" before.